source/simulator/melt.cc

```diff
@@ -1221,15 +1221,7 @@ namespace aspect
 
       LinearAlgebra::PreconditionAMG preconditioner;
       LinearAlgebra::PreconditionAMG::AdditionalData Amg_data;
-#ifdef ASPECT_USE_PETSC
-      Amg_data.symmetric_operator = false;
-#else
-      // Amg_data.constant_modes = constant_modes;
-      Amg_data.elliptic = true;
-      Amg_data.higher_order_elements = false;
```

Comment:
Actually, you want to keep these lines! You want to get rid of the ifdef branch because ASPECT no longer supports PETSc, but we still need the else branch.

Reply:
Oh okay, so I should add back in lines 1226 to 1231 ?